Automattic/wp-calypso
# Add/offer step in cancellation flow

#### Proposed Changes

* This diff adds an offer step in the cancellation flow for Jetpack products. The offer step presents a special discount for the product that is being cancelled, if one is available.
* This diff does not provide a way to take action on the offer - that will be handled in a subsequent PR.
* The offer step is set to be enabled in the development and stage environments.

#### Testing Instructions

* To test, you will need a site that has a Jetpack Scan purchase.
* Check out this branch in your test environment.
* Navigate to `/me/purchases`.
* Select your scan purchase from the purchases list.
* From the purchase screen, Click to remove/ cancel your purchase and start the cancellation flow.
* Follow the steps in the cancellation flow, but don't remove the purchase.
* After the survey question step, the next step should show a special offer, it should look similar to this:
![Screen Shot 2022-07-07 at 12 22 28 PM](https://user-images.githubusercontent.com/18016357/177872515-2eb89fce-4f49-4831-9414-178ef5cbdc0a.png)

* Confirm that the text makes sense and is set up to be properly translated.
* Confirm that the price shown for the discounted renewal rate compared to the current subscription price represents the percentage discount shown on the offer card.
